src/controller/CHIPDeviceController.h

```diff
@@ -238,6 +238,30 @@ class DLL_EXPORT DeviceController : public AbstractDnssdDiscoveryController
         return CHIP_NO_ERROR;
     }
 
+    /**
+     * This function finds the device corresponding to deviceId, and establishes
+     * a CASE session with it.
+     *
+     * Once the CASE session is successfully established the `onConnectedDevice`
```

Comment:
There is no "onConnectedDevice" here.  I know you copy/pasted this from a place that also has it wrong, but let's just fix both places instead of making them both wrong.

Reply:
Acked